sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -173,7 +173,9 @@ def plot_partial_dependence(
         differences between the `'brute'` and `'recursion'` method.
 
     n_jobs : int, default=None
-        The number of CPUs to use to compute the partial dependences.
+        The number of CPUs to use to compute the partial dependences. Computation
+        is parallelized over specified by the `features` parameter.
```

Comment:
I feel that we should avoid 'target' since that might be confused with 'label'

Reply:
I agree. I think replacing the "target features" in the docstring with "selected features" would be clearer:

```suggestion
        is parallelized over the selected features.
```

(I am trying to prevent the reader from thinking that we are computing the PD over **all** the features)

Reply:
I think referencing the `features` parameter is still appropriate, though, right? Why would you like to drop the specific reference to `features` as a parameter?

Reply:
The alternative was:

> Computation is parallelized over features specified by the `features` parameter.

which works as well.

Reply:
Agreed.